src/mount.ts

```diff
@@ -26,6 +26,7 @@ import {
   MOUNT_PARENT_NAME
 } from './constants'
 import { stubComponents } from './stubs'
+import { isString } from './utils'
```

Comment:
Maybe we can use `isString` from lodash. You would need to update the `rollup.config.js` too.